src/lib/commands.js

```diff
@@ -228,53 +228,59 @@ const createCommands = (store, cmpManager) => {
 	 * Trigger the consent tool UI to be shown
 	 */
 	const showConsentTool =  (callback = () => {}) => {
-		cmpManager.addEventListener('cmpReady', () => {
-			store.updateSection();
-			store.toggleConsentToolShowing(true);
+		cmpManager.addEventListener('cmpReady', ({data: {success}}) => {
```

Comment:
Do we need such nested structure here? Perhaps boolean would do the job?

Reply:
You are right. But what if in the future we will decide to pass additional parameter here? If we resign from nesting structure we would have to make API breaking change in such case. Furthermore it's good to have `data` value always of the same type - `Object`.

Reply:
After some consideration I agree. Probably we never want to pass additional property here besides `success`.